tests/network/sriov.go

```diff
@@ -102,6 +103,7 @@ var _ = Describe("[Serial]SRIOV", func() {
 			var err error
 			vmi, err = createAndWaitVMI(vmi)
 			Expect(err).ToNot(HaveOccurred())
+			DeferCleanup(deleteVMI, vmi)
```

Comment:
Lets say the VM failed to be deleted, will we see the error when there is only return there?
If not, would Expect would help ?

Reply:
It creates a dynamic `AfterEach`, so the answer is yes.

Reply:
but there is just a return in that dynamic `AfterEach`, don't we need `Expect` so it will be visible ?
(talking about the last return, not the panic)